tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source.ts

```diff
@@ -84,7 +84,9 @@ export class QueryParamsFeatureFlagDataSource extends TBFeatureFlagDataSource {
 
     // When media query matches positively, it certainly means user wants it but
     // it is not definitive otherwise (i.e., query params can override it).
-    if (enableDarkMode) {
+    // TODO(stephanwlee): enable the feature when most of the UI is actually
+    // ready for usage.
+    if (enableDarkMode && false) {
```

Comment:
Optional: alternatively, would it be preferred to set featureFlags.enableDarkMode to true only when both the queryParam asks for it AND the media query prefers it?  That way, it remains off by default but anyone can still test out dark theme in TB nightly as well.

Reply:
It would be impossible for me to try out and debug the feature myself :) Sure, I can use CDT's feature to mock it but it has been a bit annoying.